spec/system/switching_path_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe 'Switching Paths', type: :system do
+  let!(:default_path) { create(:path, title: 'Foundations', default_path: true) }
+  let!(:unselected_path) { create(:path, title: 'Rails') }
+
+  before do
+    sign_in(create(:user, path: default_path))
+  end
+
+  context 'on the paths index page' do
+    it 'allows a user to switch paths' do
+      visit paths_path
+
+      expect(find(:test_id, 'Foundations').value).to eq('Resume')
+      expect(find(:test_id, 'Rails').value).to eq('Switch')
+
+      find(:test_id, 'Rails').click
+
+      expect(page).to have_css '.alert-success'
+      expect(find(:test_id, 'flash')).to have_text 'You have switched to the Rails path'
+    end
+  end
+
+  context 'on the path show page' do
+    it 'allows a user to switch to that path' do
+      visit path_path(unselected_path)
+
+      expect(find(:test_id, 'Rails').value).to eq('Switch Path')
```

Comment:
I think we might able to remove this assertion since the finding and clicking the button implicitly checks the for the buttons existence. But there might be another good reason for keeping this I'm not seeing?

Reply:
I think this just leaked out of the previous test where I wanted to actually check the buttons contents based on whether the user was enrolled in that one or not. Agree it doesn't hold much value here.